specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimversionsets.json

```diff
@@ -0,0 +1,383 @@
+                            "$ref": "#/definitions/UserUpdateParameters"
+                        },
+                        "description": "Update parameters."
+                    },
+                    {
+                        "name": "If-Match",
+                        "in": "header",
+                        "required": true,
+                        "description": "The entity state (Etag) version of the user to update. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "The Api Version Set details were successfully updated."
+                    },
+                    "default": {
+                        "description": "*** Error Responses: ***\n\n * 4XX Error response giving details why the operation failed.",
+                        "schema": {
+                            "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "ApiVersionSets"
+                ],
+                "operationId": "ApiVersionSet_Delete",
+                "description": "Deletes specific Api Version Set.",
+                "x-ms-examples": {
+                    "ApiManagementDeleteApiVersionSet": {
+                        "$ref": "./examples/ApiManagementDeleteApiVersionSet.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionSetIdParameter"
+                    },
+                    {
+                        "name": "If-Match",
+                        "in": "header",
+                        "required": true,
+                        "description": "The entity state (Etag) version of the Api Version Set to delete. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "The user details were successfully deleted."
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ApiVersionSetContractProperties": {
+            "description": "An API Version Set contains the common configuration for a set of API Versions relating ",
+            "properties": {
+                "id": {
```

Comment:
>id [](start = 17, length = 2)

`id` not needed here
add `displayName`

Reply:
for `displayName` can you add validation
minLength:1
maxLength:100 

---
In reply to: [167273673](https://github.com/darrelmiller/azure-rest-api-specs/pull/3#discussion_r167273673) [](ancestors = 167273673)